BigImageViewer/src/main/java/com/github/piasy/biv/view/BigImageView.java

```diff
@@ -156,11 +159,22 @@ protected void onFinishInflate() {
 
         setOptimizeDisplay(mOptimizeDisplay);
         setInitScaleType(mInitScaleType);
+        setTapToRetry(mTapToRetry);
```

Comment:
Why do we need this?

Reply:
So that user can set it not only in xml but through public function too.